cupy/_indexing/generate.py

```diff
@@ -472,10 +472,63 @@ def mask_indices(n, mask_func, k=0):
 # TODO(okuta): Implement mask_indices
 
 
-# TODO(okuta): Implement tril_indices
+def tril_indices(n, k=0, m=None):
+    """Returns the indices of the lower triangular matrix.
+    Here, the first row contains row coordinates of all indices and
+    the second row contains column coordinates.
```

Comment:
I read "row" is used for another meaning in this sentence. How about "element" to indicate members of a tuple?

```suggestion
    Here, the first element contains row coordinates of all indices and
    the second element contains column coordinates.
```

Reply:
Sure, if you say. But I think it may create confusion, as `element` refers to a single term. What about `group of elements` instead of that?

Reply:
Perfect.